src/OpenTelemetry.Instrumentation.AspNetCore/Implementation/AspNetCoreInstrumentationEventSource.cs

```diff
@@ -43,7 +43,7 @@ public void NullPayload(string handlerName, string eventName)
             this.WriteEvent(1, handlerName, eventName);
         }
 
-        [Event(2, Message = "Request is filtered out.", Level = EventLevel.Verbose)]
+        [Event(2, Message = "Request is filtered out. EventName: '{0}'.", Level = EventLevel.Verbose)]
```

Comment:
We should add the `EventName` for `RequestFilterException` and `EnrichmentException` events as well.

Reply:
I'll open a follow-up to the follow-up for that 😄